default/python/AI/freeorion_tools/_freeorion_tools.py

```diff
@@ -311,6 +313,28 @@ def get_species_tag_grade(species_name, tag_type):
     return get_ai_tag_grade(species.tags, tag_type)
 
 
+@cache_for_session
+def get_species_tag_value(species_name: str, tag_type: AIDependencies.Tags) -> float:
+    """Get a numeric value for a skill-like (GOOD, BAD...) Tag."""
+    grade = get_species_tag_grade(species_name, tag_type)
+    if tag_type == AIDependencies.Tags.INDUSTRY:
```

Comment:
See my suggestion about function instead of the dicts.  Basically, you won't need this wrapper and could import what you need. 

As an alternative solution we could put all `AIDependencies.*_MODIFIER` to dict {AIDependencies.Tags.X, {...} } and expose only that function.